README.md

```diff
@@ -1,88 +0,0 @@
-# Spryker OS - Reference repository for DevVM
```

Comment:
Removing readme is not valid

Reply:
will add later an updated one like in release file

Reply:
added